src/host_vm.rs

```diff
@@ -4,7 +4,7 @@
 
```

Comment:
Aren't we overloading the term "Host" with another meaning (as 'host the host' in the commit message implies). I.e., not only host vm but also host_context, meaning essentially the generalized parent of a vcpu?